src/shared/ui/NavBack/index.tsx

```diff
@@ -0,0 +1 @@
+export { NavBack };
```

Comment:
```suggestion
export { NavBack } from './NavBack';
```
`NavBack`파일에서도 `default` keyword제거가 필요해요. 

혹시, 유진이 파일에서는 이게 에러가 안잡히나요? 
저는 PR올리기 전에 
1. `npm run build`를 한 번 실행해서 오류를 체크하고
2. file change도 확인해서 잘못 들어간 변경사항이 있는지 확인하는 편 이에요


유진이 PR에서도 컴파일에러가 종종 발견될때가 있어서 npm run build 실행해주면 리뷰시간이 더 짧아질 것 같아용 참고해주세요